src/govuk/components/notification-banner/_index.scss

```diff
@@ -0,0 +1,87 @@
+@include govuk-exports("govuk/component/notification-banner") {
+  /* stylelint-disable */
+
+  .govuk-notification-banner {
+    @include govuk-font($size: 19);
+    border: $govuk-border-width solid $govuk-brand-colour;
+
+    &:focus {
+      outline: 4px solid $govuk-focus-colour;
+    }
+  }
+
+  .govuk-notification-banner--success {
+    border-color: govuk-colour("green");
+  }
+
+  .govuk-notification-banner--error {
+    border-color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__header {
+    padding: 0 govuk-spacing(3) govuk-spacing(1);
+    color: govuk-colour("white");
+    background-color: $govuk-brand-colour;
+
+
+    @include govuk-media-query($from: tablet) {
+      padding: (govuk-spacing(2) - $govuk-border-width) govuk-spacing(4) govuk-spacing(2);
+    }
+  }
+
+  .govuk-notification-banner--success .govuk-notification-banner__header {
+    background-color: govuk-colour("green");
+  }
+
+  .govuk-notification-banner--error .govuk-notification-banner__header {
+    background-color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__title {
+    @include govuk-font($size: 19, $weight: bold);
+    margin: 0;
+    padding: 0;
+  }
+
+  .govuk-notification-banner__content {
+    @include govuk-font($size: 24, $weight: bold);
+    padding: govuk-spacing(3);
+
+    @include govuk-media-query($from: tablet) {
+      padding: govuk-spacing(4);
+    }
+  }
+
+  .govuk-notification-banner__link {
+    @include govuk-typography-weight-bold;
+
+    &:link,
+    &:visited,
+    &:hover,
+    &:active {
+      color: govuk-colour("blue");
+    }
+  }
+
+  .govuk-notification-banner__link--success {
+    &:link,
```

Comment:
I think it'd be worth creating helpers for 'error' and 'success' link styles in `helpers/links`. We can then update error summary to use them as well.

It also seems like it might make sense to add a `$govuk-success-colour` variable?